content/blog/stable-arvo.md

```diff
@@ -0,0 +1,98 @@
++++
+title = "Stable Arvo"
+date = 2019-11-14
+description = "In order to get closer to an Urbit that can last forever, we set for ourselves the goal of getting Arvo to a point that we can credibly call ‘stable’ by the end of 2019. "
+aliases = ["/posts/essays/stable-arvo", "/posts/stable-arvo"]
```

Comment:
Aliases are only necessary if the post was at a previous place!

Reply:
AH, gotcha — very good to know